### PR TITLE
EASYOPAC-912 - Rename files, resave existing events.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**

### CODE REVIEW

**Comment by guddo** [comment]
@NicolaeGrigoriu This hook resaves all ding_event CT nodes? This is very undesirable.

**Comment by NicolaeGrigoriu** [comment]
In order to apply the changes to the images that are in body field it is necessary to resave nodes. This was the fastest way to accomplish this.

**Review by guddo — CHANGES_REQUESTED** [review]
Manipulating nodes like this is not the way to go. It will change the edit date on all these nodes as well as require unnecessary savings. You should find another way to accomplish this.

File: modules/easyddb_dams_media_browser/easyddb_dams_media_browser.features.field_instance.inc [hunk 1]
```diff
@@ -118,10 +122,10 @@ function easyddb_dams_media_browser_field_default_field_instances() {
     'description' => '',
     'display' => array(
       'default' => array(
-        'label' => 'hidden',
+        'label' => 'above',
```
Comment (guddo) [inline modules/easyddb_dams_media_browser/easyddb_dams_media_browser.features.field_instance.inc:125]: Is this supposed to be so?

**Review by guddo — APPROVED** [review]

File: modules/easyddb_dams_media_browser/easyddb_dams_media_browser.features.field_instance.inc (hunk 1 above)
Comment (NicolaeGrigoriu) [reply modules/easyddb_dams_media_browser/easyddb_dams_media_browser.features.field_instance.inc:125]: It doesn't really matter since the field is hidden

**Comment by guddo** [comment]
@NicolaeGrigoriu Remember to move to https://github.com/ding2/ding2/pull/281 - no QA needed.

**Pull request merged**